init.c

```diff
@@ -147,26 +147,11 @@ static int other_mounts()
 static pid_t waitpids(int *wstatus)
```

Comment:
This function is so small, maybe we can just remove it and call `waitpid(-1....` inside of the handler

Reply:
done